doc/rst/tools/mason/mason.rst

```diff
@@ -311,6 +311,86 @@ If no query is provided, all packages in the registry will be listed.
 
     Packages will be listed regardless of their chplVersion compatibility.
 
+External Dependencies
+=====================
+Mason allows for specification of external, non-Chapel dependencies through pkg-config.
+An example of using the Linear Algebra library that requires both BLAS and Lapack:
+
+``Mason.toml``
+
+.. code-block:: text
+
+    [brick]
+    name = "cholesky"
+    version = "0.1.0"
+    chplVersion = "1.18.0"
+    compopts = "--ccflags -Wno-enum-conversion --ccflags -Wno-strict-prototypes"
+
+    [dependencies]
+    
+    [external]
+        [external.pkgconfig]
+        lapack = "3.8.0"
+        openblas = "*"
+
+
+This ``Mason.toml`` is updated to reflect the necessary dependency information for Mason
+to build the package in the ``Mason.lock``. Mason gets external dependency information,
+in this case, from the ``.pc`` files found via the ``PKG_CONFIG_PATH``. The ``*`` means
```

Comment:
Design Q: Should we attempt to support the range syntax for versions in addition to (or instead of) `*`, similar to `chplVersion`? e.g.

```
openblas = "2.3.." # 2.3.0 +
openblas = "4.."   # 4.0.0 +
openblas = "1..3" # 1.0.0 - 3.*.*
```

I'm OK with this being determined beyond this PR, but wanted to mention it.

Reply:
I would like to see this functionality for external deps not from pkg-config, but for this PR and the purpose of Pkg-config prototyping I think the * is more than enough user functionality.